LongWarOfTheChosen/Localization/LW_AlienPack_Integrated/LW_AlienPack_Integrated.chn

```diff
@@ -1,4 +1,4 @@
-﻿
+
```

Comment:
What's with the whitespace-only changes, lile this one? I see them scattered throughout the PR.

Reply:
> What's with the whitespace-only changes, like this one? I see them scattered throughout the PR.

That should be a change in the file encoding, from UTF-8 with BOM to UTF-8. Would this cause any bugs?

Reply:
UTF-8 is the correct one. I'll check this later, but sounds like a plausible explanation.

Reply:
> UTF-8 is the correct one. I'll check this later, but sounds like a plausible explanation.

I've checked the official repository, and it seems that most of the translation files are currently in UTF-8 encoding, with a very small portion in UTF-8 with BOM. Not sure why there is this difference, perhaps to prevent garbled characters from displaying?

Reply:
I've now checked this on my local box and UTF-8 BOM being converted to UTF-8 indeed seems to be the cause. UTF-8 without BOM has been the way of this repo, so fixes to encodingsfixes are much appreciated.

According to online sources, UTF-8 with or without BOM only have the few bytes of difference in the beginning and otherwise behave identically. Most applications should be able to handle both, so garbled characters are unlikely to be caused by that.